src/libs/actions/IOU.ts

```diff
@@ -9255,7 +9255,19 @@ function getIOUReportActionToApproveOrPay(chatReport: OnyxEntry<OnyxTypes.Report
         // This will be fixed as part of https://github.com/Expensify/Expensify/issues/507850
         // eslint-disable-next-line deprecation/deprecation
         const policy = getPolicy(iouReport?.policyID);
-        const shouldShowSettlementButton = canIOUBePaid(iouReport, chatReport, policy) || canApproveIOU(iouReport, policy);
+
+        const canPay = canIOUBePaid(iouReport, chatReport, policy);
+        const canApprove = canApproveIOU(iouReport, policy);
+
+        // Don't show settlement button if user is expense owner but not manager
+        const isExpenseOwner = iouReport?.ownerAccountID === getCurrentUserAccountID();
+        const isManager = iouReport?.managerID === getCurrentUserAccountID();
+        const canUserPay = isExpenseOwner && !isManager ? false : canPay;
```

Comment:
Is it possible for there to be multiple users that can pay? And would this mean that if someone can pay, but isn't the manager, they would see a GBR? If so, that seems wrong to me. I would think that only the manager should see the GBR while a report is awaiting approval. But that the manager could approve+pay in one go if they are the payer on the policy also.